### PR TITLE
Add embed link below each embed preview

### PR DESCRIPTION
**Pull request by faihegberg**
This PR adds a corresponding embed link under each embed preview on the "Tools" page. This is a work-in-progress. Some notes regarding necessary revisions are attached to specific parts of the code below.

### CODE REVIEW

File: src/components/Markdown/Markdown.style.ts [hunk 1]
```diff
@@ -161,5 +161,7 @@ export const Caption = styled.figcaption`
 
 export const CenterEmbed = styled.div`
   display: flex;
+  flex-direction: column;
+  align-items: center;
```
Comment (faihegberg) [inline src/components/Markdown/Markdown.style.ts:165]: Added these as each embed link text is nested under `<CenterEmbed>` in `MarkdownLink.tsx`, which without these new properties will make the text sit to the right of the embed preview box.

File: src/components/Markdown/MarkdownLink.tsx [hunk 2]
```diff
@@ -26,6 +30,36 @@ const scrollWithTimeout = (element: any, offset: number) => {
   }, 400);
 };
 
+const CanEmbed: React.FC<{ href: string }> = ({ href }) => {
+  const [isOpen, setIsOpen] = useState(false);
+  const region =
+    href === 'https://covidactnow.org/embed/us/fips/19100'
+      ? regions.findByFipsCode('19100')!
+      : undefined;
```
Comment (faihegberg) [inline src/components/Markdown/MarkdownLink.tsx:38]: This is hacky. As the "Tools" page is not location specific, the embed preview and link which are more location specific is currently set to Texas. The way to identify which location the embed preview is referring to is by the fips code in the link in the markdown file. To ensure the embed link will show the correct embed preview box, this is currently hardcoded -__-" A better solution is welcome!

File: src/components/Markdown/MarkdownLink.tsx [hunk 3]
```diff
@@ -26,6 +30,36 @@ const scrollWithTimeout = (element: any, offset: number) => {
...
+        />
+        <EmbedPrompt>
+          <span onClick={() => setIsOpen(true)}>Embed on your website</span>
+        </EmbedPrompt>
+      </CenterEmbed>
+      <EmbedPreview
+        open={isOpen}
+        onClose={() => setIsOpen(false)}
+        region={region}
+      />
```
Comment (faihegberg) [inline src/components/Markdown/MarkdownLink.tsx:58]: Although the link is nested under `<CenterEmbed>`, the second link does not appear perfectly centered. I think this is due to the size of the location specific embed preview box directly above it.

**Review by pnavarrc — APPROVED** [review]
Looks good!

**Comment by pnavarrc** [comment]
[Embed links on tools page](https://trello.com/c/IlqWlJkt/864-embed-links-on-tools-page)

**Pull request merged**